src/Core/Manifest/ManifestFileFinder.php

```diff
@@ -251,4 +260,22 @@ public function isDirectoryIgnored($basename, $pathname, $depth)
 
         return false;
     }
+
+    private function findModuleCILib(string $basename, string $pathname, int $depth): string
```

Comment:
```suggestion
    /**
     * Retrieve the CI library for the provided pathname.
     */
    private function findModuleCILib(string $basename, string $pathname, int $depth): string
```

Technically, `$basename` is not needed here because I'm working off the assumption that you won't have a composer file in your tests folder and that we always look at the parent. I kept `$basename` in because all the other methods on the class have it.